softlayer/vm/softlayer_hardware.go

```diff
@@ -148,6 +148,8 @@ func (vm *softLayerHardware) ConfigureNetworks(networks Networks) error {
 		return bosherr.WrapErrorf(err, "Failed to unmarshal userdata from hardware with id: %d.", vm.ID())
 	}
 
+	bslcommon.RETRY_COUNT = 30
```

Comment:
The reason why to set these 2 values right before this function every time is that we can't be sure if these 2 values are changed by other functions, so for making sure the values passed in the function are our expected, I have to set them every time.